cmd/monaco/v2/legacy/integration_test_utils.go

```diff
@@ -103,7 +103,7 @@ func FailOnAnyError(errors []error, errorMessage string) {
 }
 
 func getTimestamp() string {
-	return strconv.FormatInt(time.Now().UnixNano(), 10)
+	return time.Now().Format("200601020304")
```

Comment:
```suggestion
	return time.Now().Format("20060102150405")
```
Careful, '03' for the hour would format it in 12 hour format. Also, I would at least leave the seconds in